app/config/locale/translations/ru.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "ru",
     "settings.direction": "ltr",
     "emails.sender": "Команда %s",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Верификация аккаунта",
+    "emails.verification.hello": "Здравствуйте, {{name}}",
+    "emails.verification.body": "Перейдите по ссылке, чтобы подтвердить свой адрес электронной почты.",
+    "emails.verification.footer": "Если вы не запрашивали подтверждение этого адреса, проигнорируйте это сообщение.",
+    "emails.verification.thanks": "Спасибо",
+    "emails.verification.signature": "команда {{project}}",
+    "emails.magicSession.subject": "Логин",
+    "emails.magicSession.hello": "Здравствуйте,",
+    "emails.magicSession.body": "Перейдите по ссылке, чтобы войти.",
+    "emails.magicSession.footer": "Если вы не просили войти, используя этот адрес электронной почты, проигнорируйте это сообщение.",
+    "emails.magicSession.thanks": "Спасибо",
+    "emails.magicSession.signature": "команда {{project}}",
+    "emails.recovery.subject": "Сброс пароля",
+    "emails.recovery.hello": "Здравствуйте, {{name}}",
+    "emails.recovery.body": "Перейдите по этой ссылке для того чтобы сбросить свой пароль для проекта {{project}}",
+    "emails.recovery.footer": "Если вы не запрашивали сброс пароля, проигнорируйте это сообщение.",
+    "emails.recovery.thanks": "Спасибо",
+    "emails.recovery.signature": "команда {{project}}",
+    "emails.invitation.subject": "Приглашение в команду %s по проекту %s",
+    "emails.invitation.hello": "Здравствуйте",
+    "emails.invitation.body": "Это письмо отправлено вам, потому что <b>{{owner}}</b> приглашает стать членом команды <b>{{team}}</b> в проекте {{project}}.",
```

Comment:
Using `<b>` tags is a great addition, but it may not be recommended since our baseline [English translation](https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L26) and others do not use bold lettering. We can do this in the future but let's keep it simple for the sake of this PR.